vendor/blockchain_contracts/src/bin/calculate_offsets/bitcoin/rfc003/mod.rs

```diff
@@ -0,0 +1,62 @@
+use crate::calculate_offsets::{
+    concat_path, metadata::Metadata, placeholder_config::PlaceholderConfig, Contract,
+};
+use std::ffi::OsStr;
+
+pub struct BitcoinScript {
+    bytes: Vec<u8>,
+    placeholder_config: PlaceholderConfig,
+}
+
+#[derive(Debug)]
+pub enum Error {
+    PlaceholderNotFound,
+    Hex(hex::FromHexError),
+    MalformedConfig(serde_json::Error),
+    IO(std::io::Error),
+}
+
+impl From<crate::calculate_offsets::Error> for Error {
+    fn from(err: crate::calculate_offsets::Error) -> Self {
+        match err {
+            crate::calculate_offsets::Error::PlaceholderNotFound => Error::PlaceholderNotFound,
+            crate::calculate_offsets::Error::Hex(err) => Error::Hex(err),
+        }
+    }
+}
+
+impl From<crate::calculate_offsets::placeholder_config::Error> for Error {
```

Comment:
Why do we do this instead of bringing `crate::calculate_offsets::placeholder_config` into scope and using
`impl From<placeholder_config::Error> for Error {`?